node/tests/SharedTests.ts

```diff
@@ -5989,6 +5989,147 @@ export function runBaseTests<Context>(config: {
         },
         config.timeout,
     );
+
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `lmpop test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient) => {
+                const key1 = "{key}" + uuidv4();
+                const key2 = "{key}" + uuidv4();
+                const nonListKey = uuidv4();
+                const singleKeyArray = [key1];
+                const multiKeyArray = [key2, key1];
+                const count = 1;
+                const lpushArgs = ["one", "two", "three", "four", "five"];
+                const expected = new Map<string, string[]>([[key1, ["five"]]]);
+                const expected2 = new Map<string, string[]>([
```

Comment:
Please double check that it returns a `map` not a `record`

Reply:
my local tests werent running correctly (they weren't being run) it is a record, updating